src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -572,8 +732,15 @@ subroutine ocn_vmix_implicit(dt, meshPool, diagnosticsPool, statePool, forcingPo
       !  Implicit vertical solve for momentum
       !
       call mpas_timer_start('vmix solve momentum', .false.)
-      call ocn_vel_vmix_tend_implicit(meshPool, dt, kineticEnergyCell, vertViscTopOfEdge, layerThickness, layerThicknessEdge, &
```

Comment:
@mark-petersen, does this seem reasonable to you?  I'm going to be of course testing this on the drying slope test case next.

Reply:
I'm thinking I'll merge it and then if there is a bug do another PR to keep moving quickly.